### PR TITLE
SQLi AWS Lambda tainted rules for JavaScript

### PR DESCRIPTION
**Pull request by inkz**
tainted rules to find SQLi in most popular libs for JS
* pg
* mysql, mysql2/promise, mysql2
* knex
* sequelize

### CODE REVIEW

File: javascript/aws-lambda/security/mysql-sqli.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      - pattern-inside: |
+          require('mysql')
+          ...
+      - pattern-inside: |
+          require('mysql2')
+          ...
+      - pattern-inside: |
+          require('mysql2/promise')
+          ...
+      - pattern-inside: |
```
Comment (minusworld) [inline javascript/aws-lambda/security/mysql-sqli.yaml:56]: Is this leftover from the previous rule?

**Review by minusworld — COMMENTED** [review]
Don't forget the CWEs! :)

File: javascript/aws-lambda/security/mysql-sqli.yaml (hunk 1 above)
Comment (inkz) [reply javascript/aws-lambda/security/mysql-sqli.yaml:56]: yes! thank you :+1:

**Comment by inkz** [comment]
@minusworld yep, my bad :+1:

**Review by minusworld — APPROVED** [review]

**Pull request merged**